package.json

```diff
@@ -56,6 +56,8 @@
     "redux-logger": "^2.7.4",
     "redux-persist": "^4.3.1",
     "redux-persist-transform-encrypt": "^1.0.2",
+    "remote-redux-devtools": "^0.5.12",
```

Comment:
Although the linter is yelling to have these dependencies - this really shouldn't be in `dep` and should be in `devDep`. We can do `// eslint-disable-line import/no-extraneous-dependencies` as a workaround where they're being imported